spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/pubsub/it/PubSubChannelAdaptersIntegrationTests.java

```diff
@@ -141,7 +138,7 @@ public void sendAndReceiveMessageManualAck() {
 					(AckReplyConsumer) message.getHeaders().get(GcpPubSubHeaders.ACKNOWLEDGEMENT);
 			assertThat(acker).isNotNull();
 			acker.nack();
-			message = channel.receive(1000);
+			message = channel.receive(2000);
```

Comment:
To honor slowness of the Travis container and those CI servers, in other project we agreed to use a 10 seconds to wait for a message from the channel.